### PR TITLE
Add Homepage link to 404 and improve its apology

### PR DESCRIPTION
**Pull request by rlaphoenix**
This is an enhancement or feature.

## Summary

This makes the 404 page more visually appealing as well as more functional, more of a reason to exist.

## Context

N/A

### CODE REVIEW

**Comment by rlaPHOENiX** [comment]
Preview:

image

File: docs/_pages/404.md [hunk 1]
```diff
@@ -5,4 +5,5 @@ sitemap: false
 permalink: /404.html
 ---
 
-Sorry, but the page you were trying to view does not exist.
+Sorry — we couldn't find what you were looking for.  
+[Homepage]({{ '/' | absolute_url}})
```
Comment (iBug) [inline docs/_pages/404.md:9]: Recommend:

```diff
-Sorry — we couldn't find what you were looking for.  
-[Homepage]({{ '/' | absolute_url}})
+Sorry — we couldn't find what you were looking for.
+
+[<i class="fas fa-home"></i> Return to homepage]({{ '/' | relative_url}})
```

**Review by iBug — CHANGES_REQUESTED** [review]

File: docs/_pages/404.md [hunk 2]
```diff
@@ -5,5 +5,6 @@ sitemap: false
 permalink: /404.html
 ---
 
-Sorry — we couldn't find what you were looking for.  
-[Homepage]({{ '/' | absolute_url}})
+Sorry — we couldn't find what you were looking for.
+
+[<i class="fas fa-home"></i> Return to homepage]({{ '/' | relative_url}})
```
Comment (iBug) [inline docs/_pages/404.md:10]: A small point about coding convention (spacing). Recommended change:

```diff
-[<i class="fas fa-home"></i> Return to homepage]({{ '/' | relative_url}})
+[<i class="fas fa-home"></i> Return to homepage]({{ '/' | relative_url }})
```

**Review by iBug — CHANGES_REQUESTED** [review]

**Comment by rlaphoenix** [comment]
Any further review? potential merge?

**Comment by mmistakes** [comment]
Thanks for the PR but closing this with the `wontfix` label.
The 404 page here is just an example I'm using for the demo site. It can easily be customized as you have to fit your site's needs.

**Comment by iBug** [comment]
@mmistakes Honestly I'd say it's better to merge this in. It's certainly an improvement for your docs site, and you know, there could be more 404 links from time to time as the docs are updated and whatever happens...

**Comment by mmistakes** [comment]
@iBug Don't think I agree. I'm seeing a slightly reworded error text and a home link that already exists in the masthead.

**Pull request closed without merging**